src/main/java/install/FormplayerConfigEngine.java

```diff
@@ -0,0 +1,461 @@
+/**
+ *
+ */
+package install;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.commcare.api.persistence.SqliteIndexedStorageUtility;
+import org.commcare.resources.ArchiveFileRoot;
+import org.commcare.resources.JavaFileRoot;
+import org.commcare.resources.JavaHttpRoot;
+import org.commcare.resources.ResourceManager;
+import org.commcare.resources.model.*;
+import org.commcare.resources.model.installers.LocaleFileInstaller;
+import org.commcare.resources.reference.JavaResourceRoot;
+import org.commcare.suite.model.*;
+import org.commcare.util.CommCarePlatform;
+import org.javarosa.core.io.BufferedInputStream;
+import org.javarosa.core.io.StreamsUtil;
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.FormInstance;
+import org.javarosa.core.reference.ReferenceManager;
+import org.javarosa.core.services.locale.Localization;
+import org.javarosa.core.services.storage.*;
+import org.javarosa.xml.util.UnfullfilledRequirementsException;
+import org.javarosa.xpath.XPathMissingInstanceException;
+import util.PrototypeUtils;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Hashtable;
+import java.util.Vector;
+import java.util.zip.ZipFile;
+
+
+/**
+ * @author ctsims
```

Comment:
seems like it might be, if it is, is there no way to use it as a library? this means we have to keep both up to date